bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiClient.java

```diff
@@ -775,7 +775,7 @@ private boolean addInputStatus(ShellySettingsStatus status, @Nullable Shelly2Inp
     protected Shelly2RpcBaseMessage buildRequest(String method, @Nullable Object params) throws ShellyApiException {
         Shelly2RpcBaseMessage request = new Shelly2RpcBaseMessage();
         request.id = Math.abs(random.nextInt());
-        request.src = thingName;
+        request.src = "openhab-" + config.localIp; // thingName;
```

Comment:
This comment seems confusing, does it serve a purpose?
```suggestion
        request.src = "openhab-" + config.localIp;
```

Reply:
fixed